### PR TITLE
Project Popup Window

### PR DESCRIPTION
**Pull request by mrsamsharma**
- Created Temporary Popup with overlay background for mobile and desktop for reference to write javascript fucntion.
- Created a ``displayPopup()`` function which dynamically creates HTML when the user clicks on "See Project" Button
- Used loops to iterate through id's of button so that the content is specific to the project
- Finally, Event listeners are used to see user's interaction with the webpage and when the user click on the "See Project" Button it pop ups and when the user clicks or taps on the cross icon the HTML is deleted!

### CODE REVIEW

File: Js/app.js [hunk 1]
```diff
@@ -11,4 +73,103 @@ crossIcon.addEventListener('click', toggleMenu);
...
+  btnWrapper.className = 'popBtn';
+  const liveBtn = document.createElement('a');
+  liveBtn.className = 'pop-card-button';
+  liveBtn.setAttribute('href', data.seelive);
+  liveBtn.setAttribute('target', '_blank');
+  liveBtn.textContent = 'See Live ';
+  const liveBtnIcon = document.createElement('img');
+  liveBtnIcon.setAttribute('src', './images/icons/github-icon.svg');
+  liveBtn.appendChild(liveBtnIcon);
+  btnWrapper.appendChild(liveBtn);
```
Comment (codepantha) [inline Js/app.js:147]: The popup design doesn't follow the Figma design specs and cuts off some vital information.

- [ ] Please fix the 'See Live' and 'See Source' buttons not showing on both desktop and mobile devices🙏 
- [ ] Please increase the width of the popup to match the Figma design
### Your Design
image

### Figma Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi team 👋,

## STATUS: REQUESTED CHANGES ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights ⭐ 🎯 
✔️ Descriptive PR
✔️Meaningful commit messages
✔️ Linter checks
✔️ Professional README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Js/app.js (hunk 1 above)
Comment (mrsamsharma) [reply Js/app.js:147]: Hello @codepantha !
The buttons are visible, can you please tell me if they are overflowing from the screen you're reviewing form or are they not visible at all? As I can check through the live link, the buttons are visible and working perfectly!

Comment (codepantha) [reply Js/app.js:147]: Hi @samWfelice 👋 
They are neither visible on chrome nor firefox browsers

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @samWfelice, and @Kirabo19👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**